library/ui-strings/src/main/res/values/strings_sc.xml

```diff
@@ -237,6 +237,9 @@
     <!-- SC-TMP, until merged upstream: https://github.com/vector-im/element-android/pull/5781 -->
     <string name="settings_call_ringtone_use_default_stun_title">Allow fallback call assist server</string>
     <string name="settings_call_ringtone_use_default_stun_summary">Will use %s as assist when your homeserver does not offer one (your IP address will be seen by the stun server during a call
+
+    <string name="call_notification_open_app_action">Open App</string>
+    <string name="labs_enable_group_call_notifications_summary">Ring for group calls</string>
 )</string>
```

Comment:
You added the strings within another string, that's why the build can't find them

Reply:
Got it, thanks !

Reply:
@SpiritCroc , just in case if that got slipped away unnoticed - strings [has been adjusted](https://github.com/SchildiChat/SchildiChat-android/pull/250/commits/00668f80d7477de9efa2b3e3bf2de77cc0b329e5) 

Please, let us know if any other obstacles or things to be adjusted / improved!

Reply:
I'm aware of that, thanks. The main thing I'm lacking right now is time to properly test and review, I assume earliest in two week. I usually don't do video calls on mobile, but until https://github.com/element-hq/element-android/issues/8885 is fixed it's probably not too meaningful to get this released anyway.

Reply:
Thank you for a quick response and sharing what's on the plate.

We didn't have / tested video calls in mind as well - do you believe that issue [8885](https://github.com/element-hq/element-android/issues/8885) is a blocker for audio calls as well?

Reply:
If it uses the same foreground permission I assume it will be the same, I would assume both use similar implementations but I haven't checked